www.foia.gov/_api/agencies/gsa.md

```diff
@@ -59,10 +59,10 @@ data:
       label: Request description
       type: textarea
       required: true
-    - name: request_fee_waiver
+    - name: fee_waiver
       label: Request fee waiver
       type: radio
```

Comment:
Not sure what these are for exactly, but just a heads up that we have `fee_waiver` and `expedited_processing` as select fields with yes/no options.

Reply:
Yeah, these represent metadata examples... but they don't really reflect that the metadata is actually coming from the json-api and looks very different. I think I should remove these...